compiler/src/java_plugin/cpp/java_generator.h

```diff
@@ -52,7 +52,7 @@ string ServiceClassName(const google::protobuf::ServiceDescriptor* service);
 // Writes the generated service interface into the given ZeroCopyOutputStream
 void GenerateService(const google::protobuf::ServiceDescriptor* service,
                      google::protobuf::io::ZeroCopyOutputStream* out,
-                     ProtoFlavor flavor);
+                     ProtoFlavor flavor, bool disable_version);
```

Comment:
can we pass `vector<pair<string, string> >* options` here? as I will add a flag for allowing deprecated